src/EventListener/AuditLogListener/AuditLogEvent.php

```diff
@@ -0,0 +1,119 @@
+<?php
+declare(strict_types=1);
+
+namespace Landingi\EventStoreBundle\EventListener\AuditLogListener;
+
+use Landingi\EventStoreBundle\Event\AccountUuid;
+use Landingi\EventStoreBundle\Event\AggregateName;
+use Landingi\EventStoreBundle\Event\AggregateUuid;
+use Landingi\EventStoreBundle\Event\CreatedAt;
+use Landingi\EventStoreBundle\Event\EventData;
+use Landingi\EventStoreBundle\Event\EventName;
+use Landingi\EventStoreBundle\Event\SourceIp;
+use Landingi\EventStoreBundle\Event\UserUuid;
+use Landingi\EventStoreBundle\Event\AccountName;
+use Landingi\EventStoreBundle\Event\UserEmail;
+use Landingi\EventStoreBundle\Event\UserRole;
+
+final class AuditLogEvent
+{
+    private EventName $name;
+    private EventData $data;
+    private AggregateName $aggregateName;
+    private AggregateUuid $aggregateUuid;
+    private CreatedAt $createdAt;
+    private AccountUuid $accountUuid;
+    private AccountName $accountName;
+    private UserUuid $userUuid;
+    private UserEmail $userEmail;
+    private UserRole $userRole;
+    private ?SourceIp $sourceIp;
+    private ?AccountUuid $subaccountUuid;
+
+    public function __construct(
+        EventName $name,
+        EventData $data,
+        AggregateName $aggregateName,
+        AggregateUuid $aggregateUuid,
+        AccountUuid $accountUuid,
+        AccountName $accountName,
+        UserUuid $userUuid,
+        UserEmail $userEmail,
+        UserRole $userRole,
+        SourceIp $sourceIp = null,
+        AccountUuid $subaccountUuid = null
+    ) {
+        $this->name = $name;
+        $this->data = $data;
+        $this->aggregateName = $aggregateName;
+        $this->aggregateUuid = $aggregateUuid;
+        $this->createdAt = new CreatedAt(new \DateTime());
```

Comment:
I pyk... klasa nietestowalna

Reply:
ja to skopiowałem z Twojego eventu :P

Reply:
XDD a ja to skopiowałem z landingi-app. I tak sie dzieje copy-paste